### PR TITLE
Milestone 3 design

### PR DESCRIPTION
**Pull request by milosvukadinovic**

### CODE REVIEW

File: src/Components/Button.js [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+      <h2>{name}</h2>
+    </div>
+  );
+};
+
+Button.propTypes = {
   name: PropTypes.string.isRequired,
+  size: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
 };
```
Comment (SunnyAsar) [inline src/Components/Button.js:25]: user Defaultprops to  set color

File: src/Components/ButtonPanel.js [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
+      <Button name="+" size="false" color="#F5913E" />
     </div>
     <div className="grid-container group-2">
-      <Button name="7" />
-      <Button name="8" />
-      <Button name="9" />
-      <Button name="X" />
+      <Button name="7" size="false" color="#E0E0E0" />
+      <Button name="8" size="false" color="#E0E0E0" />
+      <Button name="9" size="false" color="#E0E0E0" />
```
Comment (SunnyAsar) [inline src/Components/ButtonPanel.js]: there is no need to send color and wide(size) for all the buttons. use defaultprops on the button component so you do send wide for only ` button 0` and color for the `grey buttons` only and the default color can be orange.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hey @milosvukadinovic 👋 
### wonderful work so far
- I have made a couple of comments below, please make the update
- Also, setup stickler on your repo.

#### make another request when the update is done.

**Review by BalintMendli — APPROVED** [review]
Hi @milosvukadinovic ,

All good now.

### 👍 

Let's move to the next milestone.

**Pull request merged**